File: rad_models/radusergroup.js
/* jshint indent: 1 */

export default function (sequelize, _DataTypes) {
  return sequelize.define(
    "radusergroup",
    {
      username: {
        type: "VARCHAR(64)",
        allowNull: false,
        defaultValue: "",
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      groupname: {
        type: "VARCHAR(64)",
        allowNull: false,
        defaultValue: "",
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      priority: {
        type: "INT(11)",
        allowNull: false,
        defaultValue: "1",
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
      id: {
        type: "INT(11)",
        allowNull: false,
        defaultValue: null,
        primaryKey: true,
        autoIncrement: true,
        comment: null,
        foreignKey: [Object],
      },
      update_time: {
        type: "DATETIME",
        allowNull: true,
        defaultValue: null,
        primaryKey: false,
        autoIncrement: false,
        comment: null,
      },
    },
    {
      tableName: "radusergroup",
    }
  );
}

File: internet/model/radusergroup.py
from sqlalchemy.orm import Mapped, mapped_column

from . import Base


# pyre-ignore[11]
class RadiusUserGroup(Base):
    __tablename__ = "radusergroup"

    id: Mapped[int] = mapped_column(primary_key=True)
    username: Mapped[str]
    group_name: Mapped[str] = mapped_column("groupname")
    priority: Mapped[int]
    # update_time: Mapped[datetime]
